src/config/network/wpa_supplicant.md

```diff
@@ -1 +1,103 @@
+
+[wpa\_supplicant(8)](https://man.voidlinux.org/wpa_supplicant.8) is a
+daemon that manages wireless interfaces based on a
+[wpa\_supplicant.conf(5)](https://man.voidlinux.org/wpa_supplicant.conf.5)
+configuration files.
+[wpa\_passphrase(8)](https://man.voidlinux.org/wpa_passphrase.8) and
+[wpa\_cli(8)](https://man.voidlinux.org/wpa_cli.8) are frontends to
+manage the configuration files and the running service.
+
+## Configuring wpa\_supplicant
+
+First, you must create a configuration file for your interface:
+
+    $ cp /etc/wpa_supplicant/wpa_supplicant.conf /etc/wpa_supplicant/wpa_supplicant-<device>.conf
+
+### WEP Configuration
+
+For WEP configuratoin, you can just add the following lines to your
+device's `wpa-supplicant.conf` :
+
+    # Example default configuration file for wpa_supplicant.conf(5).
+    ctrl_interface=/run/wpa_supplicant
+    update_config=1
+    ctrl_interface_group=wheel
+    eapol_version=1
+    ap_scan=1
+    fast_reauth=1
+
+    # Add your networks here
+    network={
+        ssid="MYSSID"
+        key_mgmt=NONE
+        wep_key0="YOUR AP WEP KEY"
+        wep_tx_keyidx=0
+        auth_alg=SHARED
+    }
+
+### WPA-PSK encryption
+
+For `WPA-PSK` encryption, we must generate a `pre shared key` with
+[wpa\_passphrase(8)](https://man.voidlinux.org/wpa_passphrase.8). To do
+so, run the following command:
+
+    wpa_passphrase <MYSSID> <key>
+
+You must append the output to your `wpa_supplicant.conf` file like so:
+
+    wpa_passphrase <MYSSID> <key> >> /etc/wpa_supplicant/wpa_supplicant-<device_name>.conf
+
+The resulting file should look something like this:
+
+    # Example default configuration file for wpa_supplicant.conf(5).
+    ctrl_interface=/run/wpa_supplicant
+    update_config=1
+    ctrl_interface_group=wheel
+    eapol_version=1
+    ap_scan=1
+    fast_reauth=1
+    # Add your networks here.
+    network={
+        ssid="MYSSID"
+        key_mgmt=NONE
+        wep_key0="YOUR AP WEP KEY"
+        wep_tx_keyidx=0
+        auth_alg=SHARED
+    }
+
+## Running wpa\_supplicant
+
+There are two options for running
+[wpa\_supplicant.conf(5)](https://man.voidlinux.org/wpa_supplicant.conf.5).
+You may simply run [dhcpcd(8)](https://man.voidlinux.org/dhcpcd.8),
+which will start processes automatically for your wireless interfaces,
+or you can run `wpa_supplicant` manually.
+
+### Starting throught dhcpcd hooks
+
+You may run [dhcpcd(8)](https://man.voidlinux.org/dhcpcd.8), which has
+hooks to start
+[wpa\_supplicant(8)](https://man.voidlinux.org/wpa_supplicant.8)
+automatically.
+
+> See [dhcpcd](./dhcpcd.md) for more information on starting/configuring
+> dhcpcd.
+
+### Running wpa\_supplicant manually
+
+You may start
+[wpa\_supplicant.conf(5)](https://man.voidlinux.org/wpa_supplicant.conf.5)
+manually, if you are not already running
+[dhcpcd(8)](https://man.voidlinux.org/dhcpcd.8) :
+
+    $ sudo wpa_supplicant -B -i <interface_name> -c <path/to/configuration/file>
```

Comment:
I would suggest avoiding using sudo, and use a # to indicate root shell, to be consistent with the rest of the docs.

Reply:
Good catch. This issue is a great candidate for a style guide, I will keep a note of that.